tests/python/gaia-ui-tests/gaiatest/apps/base.py

```diff
@@ -122,7 +122,7 @@ def manifest_url(self):
 
     @property
     def is_displayed(self):
-        return (self.apps.displayed_app.manifest_url == self.manifest_url and
+        return (self.apps.displayed_app.manifest_url == self.manifest_url or
```

Comment:
I change it here because I get an error in is_displayed and it works when I changed the statement with 'OR' but I know it's not the best idea